pydocteur/pr_status.py

```diff
@@ -42,8 +42,12 @@ def is_pr_approved(pr):
     if pr_reviews.totalCount == 0:
         logger.info(f"No reviews for PR {pr.number}")
         return False
-    is_approved = any(review.state == "APPROVED" for review in pr_reviews)
-    logger.info(f"is_approved for PR #{pr.number} is {is_approved}")
+    is_approved = all(review.state == "APPROVED" for review in pr_reviews)
```

Comment:
Yes but no :P

I had an `all` before, but the problem is that `get_reviews()` returns also stale reviews, so this will return all reviews, current and old ones, to it will never be merged.

What we need to do is get only the latest and not stale review from each reviewer maybe use some stuff from https://pygithub.readthedocs.io/en/latest/github_objects/PullRequestReview.html

Reply:
Fixed in #45 with another approch.